phoenix-core-server/src/main/java/org/apache/phoenix/coprocessor/PhoenixMasterObserver.java

```diff
@@ -74,11 +82,19 @@ public class PhoenixMasterObserver implements MasterObserver, MasterCoprocessor
             = "UPSERT INTO " + SYSTEM_CDC_STREAM_NAME + " (TABLE_NAME, STREAM_NAME, PARTITION_ID, "
             + "PARENT_PARTITION_ID, PARTITION_END_TIME) VALUES (?,?,?,?,?)";
 
+    private static final int MAX_RETRY_COUNT = 20;
```

Comment:
Can this be in the config with default value of 20?

Reply:
We could but probably good to avoid config here. I verified that HBase will also retry when coproc throws IOE here anyways.